src/plugins/intel_gna/tests/functional/Import_export_tests/backward_compatibility/backward_compatibility.cpp

```diff
@@ -0,0 +1,229 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+
+#include <map>
+#include <memory>
+#include <vector>
+
+#include "helpers/test_model_repo.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/core/shape.hpp"
+#include "openvino/core/type.hpp"
+#include "openvino/opsets/opset9.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+
+using namespace ov::opset9;
+
+typedef std::tuple<ov::element::Type,                   // Network Precision
+                   std::string,                         // Target Device
+                   std::string,                         // Name Export Model
+                   std::map<std::string, std::string>,  // Export Configuration
+                   std::map<std::string, std::string>   // Import Configuration
+                   >
+    exportImportNetworkParams;
+
+class BackwardCompatibility : public testing::WithParamInterface<exportImportNetworkParams>,
+                              virtual public LayerTestsUtils::LayerTestsCommon {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<exportImportNetworkParams> obj) {
+        ov::element::Type input_prc;
+        std::string targetDevice;
```

Comment:
```suggestion
        std::string target_device;
```

Reply:
it cannot be changed because it is property of base class